### ISSUE TITLE
installation instructions

### ISSUE THREAD

**Issue by jens120** [SMTorg/smt#141]
Hi,
Can you please make installation instructions more informative for people not so versed in pip and the like. 
Sorry for the many questions but here they are:

- I have this error whilst installing via `$ pip install git+git://github.com/SMTorg/smt.git`. I am not sure what it means. 

```
...
copying smt/extensions\mfk.py -> build\lib.win-amd64-3.7\smt/extensions
copying smt/extensions\moe.py -> build\lib.win-amd64-3.7\smt/extensions
copying smt/extensions\vfm.py -> build\lib.win-amd64-3.7\smt/extensions
copying smt/extensions\__init__.py -> build\lib.win-amd64-3.7\smt/extensions
running build_ext
building 'smt.surrogate_models.rbfclib' extension
error: Microsoft Visual C++ 14.0 is required. Get it with "Microsoft Visual C++ Build Tools": https://visualstudio.microsoft.com/downloads/


----------------------------------------
Command "C:\Users\user1\D\anaconda3\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\user1\\AppData\\Local\\Temp\\pip-req-build-a4wor94u\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\user1\AppData\Local\Temp\pip-record-kqmnukkv\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\user1\AppData\Local\Temp\pip-req-build-a4wor94u\

```

- Why can I not just use `pip install smt` Is it because this will fetch an old version?

- I installed python via anaconda. Can I therefore use `conda install`?

- Finally https://smt.readthedocs.io/en/latest/_src_docs/getting_started.html states 

    > In terminal, navigate to the directory in which you want to download the source code and use the command git clone https://github.com/SMTorg/smt.git to download.

   should this be the ....anaconda3\Lib\site-packages directory? I ran the `pip install -e . ` command in a temporary directory and I think this created some troubles when it came to uninstalling.

Thanks in advance

**Comment by relf** [comment]
Quick answer: For windows and Python 3.7, <code>pip install smt</code> should work.

**Comment by jens120** [comment]
cheers

**Comment by relf** [comment]
Indeed there was an error in my packaging. I have just updated it. Could you try <code>pip install -U smt</code> and see if it works now?

**Comment by jens120** [comment]
Hi relf,
thank you for the prompt response. It is much appreciated.
I ran the example: https://smt.readthedocs.io/en/latest/_src_docs/surrogate_models/rbf.html again but received the following error:
```
File "W:/staff-homes/s/user1/My Documents/Surrogate/RBFtest.py", line 4, in <module>
    from smt.surrogate_models import RBF

ImportError: cannot import name 'RBF' from 'smt.surrogate_models' (C:\Users\user1\Anaconda3\lib\site-packages\smt\surrogate_models\__init__.py)


```
In the meantime I have switched to a different windows machine

The kriging example is now working and I am pretty certain it was not working before the update

**Comment by relf** [comment]
Longer answer to your questions:
> I have this error whilst installing via $ pip install git+git://github.com/SMTorg/smt.git. I am not sure what it means.

SMT contains some surrogates in C++ that have to compiled. On windows it requires MS visual studio hence the message you got.
> Why can I not just use pip install smt Is it because this will fetch an old version?

Well, we have started to package SMT on Pypi so yes <code>pip install smt</code> should work but currently not really tested (you've just started, thanks :wink:) neither documented. 
> I installed python via anaconda. Can I therefore use conda install?

As far as I am concerned, I do not maintain SMT packaging on conda. Nevertheless SMT is packaged on [conda forge](https://anaconda.org/conda-forge/smt) but for now is only available in 0.2.2. 

> Finally https://smt.readthedocs.io/en/latest/_src_docs/getting_started.html states
    In terminal, navigate to the directory in which you want to download the source code and use the command git clone https://github.com/SMTorg/smt.git to download.
should this be the ....anaconda3\Lib\site-packages directory? I ran the pip install -e . command in a temporary directory and I think this created some troubles when it came to uninstalling.

There is a formatting glitch in the doc. It means that : 
* either you download and extract an smt archive (.tar.gz or .zip)
* or you clone the repo

then you go in that newly created smt directory to do <code>pip install -e .</code>. Actually, we should update that as it is a development mode installation which is not required (and indeed can lead to trouble when you uninstall). 

As a summary, if you just want to use SMT, the way to go is  :

* either <code>pip install smt</code> to get the last tagged version
* or <code>pip install git+https://github.com/SMTOrg/smt.git</code> to install from the GitHub repo
